controllers/ssp_controller.go

```diff
@@ -239,9 +239,6 @@ func (r *sspReconciler) isRestartNeeded(sspObj *ssp.SSP) bool {
 	if reflect.DeepEqual(r.lastSspSpec, ssp.SSPSpec{}) {
```

Comment:
This func is never returning `true`, so it can likely be dropped.

Reply:
When the operator starts, `r.lastSspSpec` is a zero object, which in this case is `ssp.SSPSpec{}`. It is set in method `r.clearCacheIfNeeded()`, which is called after this `r.isRestartNeeded()`:

https://github.com/kubevirt/ssp-operator/blob/9759ddfe94f066df6cb0d97e727bf06342b6d7bd/controllers/ssp_controller.go#L151-L152

Reply:
_I've removed my comment here, because it was irrelevant._